esp-wifi/src/timer/timer_esp32.rs

```diff
@@ -22,6 +22,23 @@ pub fn setup_radio_isr() {
     }
 }
 
+pub fn shutdown_radio_isr() {
+    #[cfg(feature = "ble")]
+    {
+        interrupt::disable(crate::hal::Cpu::ProCpu, peripherals::Interrupt::RWBT);
+        interrupt::disable(crate::hal::Cpu::ProCpu, peripherals::Interrupt::BT_BB);
+
+        unwrap!(interrupt::enable(
+            peripherals::Interrupt::ETH_MAC,
+            interrupt::Priority::Priority1
+        ));
+        unwrap!(interrupt::enable(
+            peripherals::Interrupt::UART0,
+            interrupt::Priority::Priority1
+        ));
```

Comment:
would be good to double check if this is still needed in `setup_radio_isr` and here - and if it is at least have a comment since it looks strange

Reply:
Thanks for noticing! I originally did it just returning things to their original state, but now yes, I definitely see that it's not necessary. 
And actually yes, those interrupts should have been turned off by default 🤷🏼